src/dispatch/plugin/models.py

```diff
@@ -0,0 +1,66 @@
+from typing import List, Optional
+
+from sqlalchemy import Column, Integer, String, Boolean
+from sqlalchemy_utils import TSVectorType, JSONType
+
+from dispatch.database import Base
+from dispatch.models import DispatchBase
+from dispatch.plugins.base import plugins
+
+
+class Plugin(Base):
+    id = Column(Integer, primary_key=True)
+    title = Column(String)
+    slug = Column(String, unique=True)
+    description = Column(String)
+    version = Column(String)
+    author = Column(String)
+    author_url = Column(String)
+    type = Column(String)
+    enabled = Column(Boolean)
+    configuration = Column(JSONType)
+
+    @property
+    def instance(self):
+        """Fetches a plugin instance that matches this record."""
+        return plugins.get(self.slug)
+
+    search_vector = Column(TSVectorType("title"))
+
+
+# Pydantic models...
+class PluginBase(DispatchBase):
+    pass
+
+
+class PluginCreate(PluginBase):
+    title: str
+    slug: str
+    author: str
+    author_url: str
+    type: Optional[str]
```

Comment:
Shouldn't `type` be mandatory? How's the following call going to work if not `plugin_service.get_active(db_session=db_session, plugin_type="ticket")`